doc/contributing.rst

```diff
@@ -253,42 +253,98 @@ How to make a release
 
 To make a new release of the package, follow these steps:
 
-1. Check that the nightly build on CircleCI was successful
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+1. Check the tests on GitHub Actions and CircleCI
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Check the ``nightly`` `build on
-CircleCI <https://circleci.com/gh/ESMValGroup/ESMValCore/tree/master>`__.
-All tests should pass before making a release.
+Check the ``nightly``
+`build on CircleCI <https://circleci.com/gh/ESMValGroup/ESMValCore/tree/master>`__
+and the
+`GitHub Actions run <https://github.com/ESMValGroup/ESMValCore/actions>`__.
+All tests should pass before making a release (branch).
 
-2. Make a pull request to increase the version number
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+2. Increase the version number
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The version number is stored in ``esmvalcore/_version.py``,
-``package/meta.yaml``, ``CITATION.cff``. Make sure to update all files. See
-https://semver.org for more information on choosing a version number.
-
-3. Make the release on GitHub
+``package/meta.yaml``, ``CITATION.cff``. Make sure to update all files.
+Also update the release date in ``CITATION.cff``.
+See https://semver.org for more information on choosing a version number.
+Make a pull request and get it merged into ``master``.
+
+3. Add release notes
+~~~~~~~~~~~~~~~~~~~~
+Use the script
+`esmvaltool/utils/draft_release_notes.py <https://docs.esmvaltool.org/en/latest/utils.html#draft-release-notes-py>`__
+to create create a draft of the release notes.
+This script uses the titles and labels of merged pull requests since the
+previous release.
+Review the results, and if anything needs changing, change it on GitHub and
+re-run the script until the changelog looks acceptable.
+Copy the result to the file ``doc/changelog.rst``.
+Make a pull request and get it merged into ``master``.
+
+3. Create a release branch
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+Create a branch off the ``master`` branch and push it to GitHub.
+Ask someone with administrative permissions to set up branch protection
+rules for it so only you and one other person can push to it.
```

Comment:
```suggestion
rules for it so only you and the admin can push to it.
```

Reply:
That would be a bit vague, who is 'the admin'? shall I write, 'the release manager of the previous release' or 'the backup release manager' instead?

Reply:
yeps, sorry, admin is supervague yeah

Reply:
Added!